lib/util/constants.js

```diff
@@ -85,6 +85,12 @@ module.exports.iceRestartBackoffConfig = {
   randomisationFactor: 0.5
 };
 
+module.exports.reconnectBackoffConfig = {
+  factor: 1.1,
+  initialDelay: 1,
+  randomisationFactor: 0.5
+};
```

Comment:
Where were the constants derived from? The [FRD](https://code.hq.twilio.com/client/sdk-frd/blob/master/video/rsp-tcmp-wss-client.md#rsp-transport-state-machine) is not very explicit other than stating that an exponential backoff should be used.

Reply:
I used the same constants that we use for iceConnectionBackoff. We do use exponential backoff, with a factor of 1.1 - so that each attempt is at `lastAttemptDelay * 1.1`

Reply:
@ceaglest - what factor do you use in CPP sdk ?

Reply:
* 80 ms for the initial backoff
* 0.5 for randomization factor
* 1.5 backoff multiplier